### ISSUE TITLE
More on nextflow log in debugging?

### ISSUE THREAD

**Issue by ewallace** [riboviz/riboviz#378]
For the [debugging the nextflow workflow](https://github.com/riboviz/riboviz/blob/main/docs/user/prep-riboviz-run-nextflow.md#debugging), I'm wondering if it would be more helpful to talk about the `nextflow log` first?

Commands like:
`nextflow log RUN_ID -filter 'status == "FAILED"' -f name, script, stdout, stderr, env`
achieve the same thing as we document, in a less generic but perhaps more user-friendly way?

Basically the difference is that using `nextflow log` doesn't need the user to keep track of the hash directories, and the user can be less of a unix expert.

**Comment by mikej888** [comment]
* [docs/user/prep-riboviz-run-nextflow.md](https://github.com/riboviz/riboviz/blob/main/docs/user/prep-riboviz-run-nextflow.md#debugging). Complete rewrite to describe use of `nextflow log` in preference to using `work/` directly.
* [docs/developer/debugging.md](https://github.com/riboviz/riboviz/blob/main/docs/developer/debugging.md#debugging-and-nextflow). Chopped down text to remove overlap with Debugging in `prep-riboviz-run-nextflow.md`, instead providing link to that section.
* [docs/user/prep-riboviz-operation.md](https://github.com/riboviz/riboviz/blob/main/docs/user/prep-riboviz-operation.md#nextflow-work-directory). Added use of `nextflow log` to get step-specific directory

**Comment by ewallace** [comment]
That documentation is extremely clear, I approve!!!
